package.json

```diff
@@ -215,6 +215,23 @@
       "output": []
     }
   },
+  "apiVersionMetadata": {
```

Comment:
In orgUtils.ts I have provided details on what this section is and how it is used